scripts/release-notes.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import os
```

Comment:
It's been ages since I wrote Python code, so don't spare any comments. I have no strong opinion at all.

Reply:
One thing I tried hard to do is to use built-in modules so that we don't have to have a `requirements.txt` or deal with env-dependent things for such a small utility.